envoy/config/filter/http/jwt_authn/v2/config.proto

```diff
@@ -36,20 +38,27 @@ message JSONWebToken {
   // <https://tools.ietf.org/html/rfc7519#section-4.1.3>`_. that are allowed to
   // access. A JWT containing any of these audiences will be accepted.
   //
-  // Example::
+  // Example:
+  //
+  // .. code-block:: yaml
   //
   //     audiences:
   //     - bookstore_android.apps.googleusercontent.com
   //       bookstore_web.apps.googleusercontent.com
   //
   repeated string audiences = 2;
 
-  // URL of the provider's public key set to validate signature of the
-  // JWT. See `OpenID
+  // URL of the provider's public key server. The public key is used to validate
```

Comment:
jwks stands for Json Web Key Set: https://tools.ietf.org/html/rfc7517#appendix-A

Reply:
Updated the comment. Thanks